### ISSUE TITLE
Failed to provision: `Build failed: Unknown task: install_prereqs`

### ISSUE THREAD

**Issue by briandant** [appsembler/sultan#8]
My first run of `make instance.create` failed:

```
 Provisioning forum...
+ ./provision-forum.sh
++ docker-compose -f docker-compose.yml -f docker-compose-host.yml -f docker-compose-themes.yml up -d forum
edx.devstack.mongo is up-to-date
edx.devstack.elasticsearch is up-to-date
edx.devstack.memcached is up-to-date
Starting edx.devstack.forum ... 
Starting edx.devstack.forum ... done
++ docker-compose exec -T forum bash -c 'source /edx/app/forum/ruby_env && cd /edx/app/forum/cs_comments_service && bundle install --deployment --path /edx/app/forum/.gem/'
Don't run Bundler as root. Bundler can ask for sudo if it is needed, and
installing your bundle as root will break this application for all non-root
users on this machine.
Warning: the running version of Bundler (1.16.6) is older than the version that created the lockfile (1.17.3). We suggest you upgrade to the latest version of Bundler by running `gem install bundler`.
Makefile:59: recipe for target 'dev.provision.all' failed
make: *** [dev.provision.all] Error 137
Connection to 34.66.56.53 closed.
targets/instnace.mk:71: recipe for target 'instance.run' failed
make[1]: *** [instance.run] Error 2
make[1]: Leaving directory '/home/briandant/sultan'
targets/instnace.mk:28: recipe for target 'devstack.provision' failed
make: *** [devstack.provision] Error 2
```

I then re-ran `make instance.run`, which succeeded. It told me to run `make devstack.provision`. I then get this, which looks to be a very similar error:

```
++ docker-compose exec -T lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/app/edxapp/edx-platform && NO_PYTHON_UNINSTALL=1 paver install_prereqs'
Build failed: Unknown task: install_prereqs
Makefile:59: recipe for target 'dev.provision.all' failed
make: *** [dev.provision.all] Error 1
Connection to 34.66.56.53 closed.
targets/instnace.mk:71: recipe for target 'instance.run' failed
make[1]: *** [instance.run] Error 2
make[1]: Leaving directory '/home/briandant/sultan'
targets/instnace.mk:28: recipe for target 'devstack.provision' failed
make: *** [devstack.provision] Error 2
```

**Comment by ahmedaljazzar** [comment]
@amirtds We had a similar error here. I'll try to figure out STR and see what happens here.

**Comment by ahmedaljazzar** [comment]
This should be resolved in #10. The issue was in the missing `OPENEDX_RELEASE` environment variable on the remote machine.
